examples/instance-pool/main.tf

```diff
@@ -7,27 +7,28 @@ variable "template" {
 }
 
 data "exoscale_compute_template" "instancepool" {
-  zone = "${var.zone}"
-  name = "${var.template}"
+  zone = var.zone
+  name = var.template
 }
 
 resource "exoscale_instance_pool" "instancepool-test" {
   name = "terraforminstancepool"
   description = "test"
-  template_id = "${data.exoscale_compute_template.instancepool.id}"
+  template_id = data.exoscale_compute_template.instancepool.id
   service_offering = "Medium"
```

Comment:
While this is not directly related to your changes, you might as well take this global configuration update opportunity to fix this problem ;)

```console
$ cd examples/instance-pool/
$ terraform init

Initializing the backend...

Initializing provider plugins...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.

$  terraform validate

Error: "service_offering" must be lowercase, got: "Medium"

  on main.tf line 14, in resource "exoscale_instance_pool" "instancepool-test":
  14: resource "exoscale_instance_pool" "instancepool-test" {
```